spec/lib/tasks/correct_renewal_postcodes_spec.rb

```diff
@@ -0,0 +1,117 @@
+require "rails_helper"
+require "rake"
+
+RSpec.describe "correct_renewal_postcodes" do
+  describe ":correct_renewal_postcodes", type: :task do
+    subject(:task) { Rake::Task["correct_renewal_postcodes"] }
+
+    before do
+      Rake.application.rake_require("tasks/correct_renewal_postcodes")
+      Rake::Task.define_task(:environment)
+      task.reenable
+    end
+
+    context "when the rake task is run" do
```

Comment:
This looks like it wants a test for not updating non-renewal logs - that feels like the highest value thing we want to be sure of before running this.

Reply:
For sure, thanks! Added it now!